### ISSUE TITLE
Phanotate does not work properly

### ISSUE THREAD

**Issue by Dx-wmc** [gbouras13/pharokka#244]
* phrokka version: 1.2.1
* Python version: 3.7
* Operating System: linux

### Description
Dear Sir,

When I run a genome (GCA_000154865.1.fasta, 4M), it stays at the **Run Phanotate** step even though I have waited for more than 10 hours and cannot come up with results. 
Interestingly when I run another Klebsiella pneumonia genome, it normally runs (38 minutes), I am confused as to what causes this.

The command I used is `pharokka.py -i GCA_000154865.1.fasta -o test2 -t 60 -d ~/database/pharokka/pharokka_v1.2.0_databases -p tes2t -l test -f`

Could you tell me why this happened?

**Comment by gbouras13** [comment]
Hi @Dx-wmc,

Phanotate is a tool designed to predict ORFs in phage genomes & the biggest ever reported phage is just under 1 MB, so I am not too surprised to see it failing on whole bacterial genomes. 

Pharokka is also not designed to predict prophages, so running whole bacterial genomes is not recommended.

I have attempted to use it myself similarly to you to annotate an entire Staphylococcus aureus genome when I first tested Pharokka and it similarly took hours - I think it is just a case of PHANOTATE  not being used for what it is designed for.

If you want to run a whole bacterial genomes to annotate prophages through pharokka, I would recommend a few options:

1. Use a prophage prediction tool (there are very very many, I'd recommend [Phispy](https://github.com/linsalrob/PhiSpy) as being quick and easy to use), then annotate the predicted prophages with Pharokka - my number 1 recommendation and often what I do!
2. Use pharokka with prodigal (-g prodigal) - it should work.
3. Split the whole genome up into smaller contigs and re-run pharokka (especially if you know coordinates to split at that are intergenic).
4.  Use [Bakta](https://github.com/oschwengers/bakta) - it should do a decent job on the prophage regions, it's a pretty amazing program!

George

**Comment by Dx-wmc** [comment]
Hi George, 

Thank you very much for your patient answer, I have benefited a lot from it. In my continuous testing, I found that Pharokka can also run smoothly on some whole genomes, but I did not compare the corresponding output results with other tools. According to your advice, I will also improve the process of phage recognition. In short, Phantotate is an excellent software and I will keep trying it.
